client/src/partials/finance/cashflow/cashflow.html

```diff
@@ -0,0 +1,239 @@
+<div class="flex-header static">
+  <div class="bhima-title">
+    <ol class="headercrumb">
+      <li class="static">{{ "TREE.FINANCE" | translate }}</li>
+      <li class="title">
+        {{ "CASHFLOW.TITLE" | translate }}
+        <b ng-show="ReportCtrl.state !== 'generate'" class="label label-default">
```

Comment:
Additionally, in the future, instead of doing
```html
<element ng-show="value !== boolean" ...>
```
You might try 
```html
<element ng-hide="value === boolean" ...>
```
to increase readability.  But it is a style choice - whatever you feel most comfortable with!